### ISSUE TITLE
Some components are not available on NPM

### ISSUE THREAD

**Issue by omar-belkhodja-bacdev** [material-components/material-components-web-components#76]
The v0.1.2 tag integrates many components. Many of them (like mwc-menu) are not available through NPM install.

**Comment by fernandopasik** [comment]
This is the list that seems to not being published:
```
@material/mwc-card
@material/mwc-chips
@material/mwc-dialog
@material/mwc-icon-toggle
@material/mwc-linear-progress
@material/mwc-list
@material/mwc-menu
@material/mwc-sass-render
@material/mwc-select
@material/mwc-slider
@material/mwc-snackbar
@material/mwc-tabs
@material/mwc-textfield
@material/mwc-typography
```

**Comment by fernandopasik** [comment]
@omar-belkhodja-bacdev check the main [readme](https://github.com/material-components/material-components-web-components/blob/master/README.md) it says:

> :warning: These components are still a work in progress. Not all elements are available on NPM yet. We are working on the remaining elements and publish them to the NPM registry once we have a first working version. :warning:

**Comment by jmorille** [comment]
It could be nice to have the working components that was presented in the demo site....
At least for testing purpose...

**Comment by ErikHellman** [comment]
Any updates on this? It is really hard to try out the Material Components when so many things are missing.

**Comment by LostInBrittany** [comment]
Ping? I fully agree with @jmorille , having the elements of the demo site in npm registry would be really useful.

**Comment by chwzr** [comment]
Please publish theese components! If they are buggy we all can help make this components work stable. But without a published versions this will never happen.

**Comment by omarbelkhodja** [comment]
One possible solution World be to create a fork, and fix there what needs to be fixed

**Comment by web-padawan** [comment]
Please wait until Polymer/lit-element#132 lands, and get released first, as there is a significant API refactoring process ongoing.

**Comment by zevisert** [comment]
[Polymer/lit-element#132](https://github.com/Polymer/lit-element/pull/132) landed in [v0.6.0-dev.6](https://github.com/Polymer/lit-element/commits/v0.6.0-dev.6), also published on npm as [@polymer/lit-element@dev](https://www.npmjs.com/package/@polymer/lit-element/v/0.6.0-dev.6).

I've got plenty of time to spend on this. What would you (@azakus / @web-padawan) like to have done first?

**Comment by LostInBrittany** [comment]
I've also got spare time for this, and I kinda need some of these elements, so I would love to help.
What would you (@azakus @web-padawan ) want me to do? We could split some tasks with @zevisert :)

**Comment by web-padawan** [comment]
I'm not a maintainer of this repo, guys 😀
But I guess you could help somehow with the tooling, apparently repo was copied from `material-components-web` and there seem to be some leftovers, dependencies in package.json could be updated, etc.

**Comment by elf-pavlik** [comment]
All components included in demo seems to have *first working version* and semantic versioning supposed to take care of breaking changes. I hope all demoed components will get published soon, especially that `package.json` doesn't seem to have a way to handle git monorepos :disappointed:

**Comment by touhonoob** [comment]
The components are being rewritten into TypeScript. I guess they will publish new NPM packages once this pull request has been merged: https://github.com/material-components/material-components-web-components/pull/109

**Comment by elf-pavlik** [comment]
What ways do you recommend to use the unpublished components? `npm link` / `yarn link`, use `file://` in `package.json` to point to local repo (seems to make sense only if single person does the development) any other options?

**Comment by mkay581** [comment]
@elf-pavlik I think just publishing the components makes the most sense. Is there any reason why the current components in this repo just can't be published to npm now? they may not be in the best state, but it would make debugging and contributing much easier for people like myself who want to help out.

**Comment by leegee** [comment]
And another?

`Couldn't` find package "@material/mwc-icon" on the "npm" registry.`

**Comment by bennypowers** [comment]
`@material/mwc-icon-button` also not available

**Comment by motss** [comment]
Still awaiting `@material/mwc-icon-button` to get published. `paper-icon-button-light` is too heavy for my use case.

**Comment by azakus** [comment]
Our plan for components not yet released is to have them design reviewed by the Material Design team.
Not all of the components are ready to meet those specifications, and we will release them only after they are ready.

Thanks for your patience!

**Comment by sunilshah123** [comment]
> Our plan for components not yet released is to have them design reviewed by the Material Design team.
> Not all of the components are ready to meet those specifications, and we will release them only after they are ready.
> 
> Thanks for your patience!

I have been waiting for a few months for these components to appear on NPM. Do you have an ETA? Thanks.

**Comment by elyobo** [comment]
Oddly I'm seeing something like the opposite, `@material/mwc-chips` is on NPM but doesn't appear to exist in the current master (although, weirdly, there are _tests_ for it...)
